frontend/test/test_oqd/oqd/oqd_beam_parameters.toml

```diff
@@ -0,0 +1,17 @@
+oqd_config_schema = "v0.1"
```

Comment:
I created these TOML files for testing since the values in the "production" version in the src directory are likely going to get frequent updates from OQD. It would be a large burden to have to update the values in the test cases to keep them in sync with the values in the production TOML files. The values in the test files are not important, so this way we can more easily test that the files are being loaded correctly without worrying about changes to the values stored within them.

Reply:
I see, thanks for the explanation!